problems/removeVowels.js

```diff
@@ -6,8 +6,24 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+const removeVowels = (str) => {
+    let newStr = ""
+    for(let i =0; i < str.length;i++){
+        if(str.length === aeiouAeiou){
+           return str.replace(/ aeiouAEIOU/g, "")
```

Comment:
This is RegEx and not something you should use at this point unless you fully understand it and could replicate it in an interview.